### PR TITLE
Fix maven project assembly failure

### LINKED ISSUE #24: Compilation failure due to web.xml missing
Maven project assembly stage fails due to web.xml missing.

### PR DESCRIPTION
**Pull request by MagoDelBlocco**
Closes #24 
Added a property to pom.xml to bypass web.xml verification.

AFAIK, no web.xml was provided in the project's structure from the get-go.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,9 @@
...
 .idea
+step240-2020.iml
 .DS_Store
 target/
 webapp/.cache
 webapp/dist
-webapp/node_modules
\ No newline at end of file
+webapp/node_modules
+
```
Comment (MagoDelBlocco) [inline .gitignore:9]: nit: Remove extra newline here and in index.html

File: .gitignore [hunk 2]
```diff
@@ -1,7 +1,9 @@
 # Please keep this list in ASCII sort order
 .idea
+step240-2020.iml
```
Comment (samuelsogbesan) [inline .gitignore:3]: Please move items to be in [ASCII order](http://support.ecisolutions.com/doc-ddms/help/reportsmenu/ascii_sort_order_chart.htm). (For the most part, it is alphabetical)

**Comment by samuelsogbesan** [comment]
@holounic Unless the files there are necessary, I suggest clearing the main/java/webapp folder since that will be where the files for #22 are going (React Web App Files). I see a potential merge conflict here otherwise.

**Review by samuelsogbesan — CHANGES_REQUESTED** [review]
Please reorder the addition to .gitignore. 
Approval also pending discussion on /webapp, but that may be out of the scope of this PR.

File: .gitignore (hunk 2 above)
Comment (krasimir-google) [inline .gitignore:3]: nit: (see comment at line 1) please keep this list in ASCII sort order

File: src/main/webapp/example.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Example</title>
+  </head>
+  <body>
+    <h1>You was redirected to example.html via servlet</h1>
```
Comment (krasimir-google) [inline src/main/webapp/example.html:8]: nit: ... You **were** redirected ...

Comment (holounic) [reply src/main/webapp/example.html]: oops

File: target/step240-2020-1.0/WEB-INF/appengine-web.xml [hunk 4]
```diff
@@ -1,10 +1,10 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (krasimir-google) [inline target/step240-2020-1.0/WEB-INF/appengine-web.xml:1]: To follow-up:
I believe no files under target/ should be submitted (see the .gitignore file).
Feel free to revert these changes in this PR and create a new PR that deletes the target/ directory.

**Review by samuelsogbesan — APPROVED** [review]
LGTM, as suggested /target should be removed in a later PR.

**Review by krasimir-google — APPROVED** [review]
Thank you!

**Pull request merged**